pkg/printers/name.go

```diff
@@ -109,11 +109,11 @@ func printObj(w io.Writer, name string, operation string, dryRun bool, groupKind
 	}
 
 	dryRunMsg := ""
-	if dryRun {
```

Comment:
I'd leave this code as it was before.

Reply:
@soltysh moved "dryRun" bits out of the name printer. It's now handled by the ToPrinter method.